app/controllers/items_controller.rb

```diff
@@ -45,9 +45,10 @@ def update
 
   def destroy
     item = Item.find(params[:id])
-    Review.where(item_id: item.id).destroy_all
+    # Review.where(item_id: item.id).destroy_all
```

Comment:
Commented out could delete since Review has dependence to Item

Reply:
![](https://media3.giphy.com/media/kHO1jmC4gEb3jy1lpf/giphy.gif?cid=5a38a5a2a7eff760c93cc6a071ae7a00b71bbdb36567605a&rid=giphy.gif)